checkstyle-tester/diff.groovy

```diff
@@ -209,7 +212,7 @@ def generateDiffReport(reportsDir, masterReportsDir, patchReportsDir, baseConfig
                     def baseReport = "$masterReportsDir/$projectName/checkstyle-result.xml"
                     def patchReport = "$patchReportsDir/$projectName/checkstyle-result.xml"
                     def outputDir = "$reportsDir/diff/$projectName"
-                    def diffCmd = "java -jar $diffToolJarPath --baseReport $baseReport --patchReport $patchReport --output $outputDir --baseConfig $baseConfig --patchConfig $patchConfig"
+                    def diffCmd = "java -jar $diffToolJarPath --baseReport $baseReport --patchReport $patchReport --output $outputDir --baseConfig $baseConfig --patchConfig $patchConfig --shortFilePaths $shortFilePaths"
```

Comment:
[The option is either on or off. It doesn't accept a parameter.](https://github.com/checkstyle/contribution/blob/master/patch-diff-report-tool/src/main/java/com/github/checkstyle/Main.java#L279)
I was expecting it to just build the string like `launch.groovy` does for `ignoreExceptions`.